Remotion/Integration/Domain/Integration.Domain.csproj

```diff
@@ -29,10 +29,5 @@
     <ProjectReference Include="..\..\Validation\Globalization\Validation.Globalization.csproj" />
     <ProjectReference Include="..\..\Validation\Mixins\Validation.Mixins.csproj" />
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="Remotion.IO" />
-    <PackageReference Include="Remotion.IO.Archive.Zip" />
-    <PackageReference Include="Remotion.TypePipe" />
```

Comment:
Was TypePipe also an old reference too?

Reply:
@Frazerus yes/no. it's a trnasitive dependency and there's no good reason why it's (still) in there. since I was touching those lines and the change is transparent, I didn't want to make a separate PR for this cleanup.